(DOCS-857) Update benchmark docs with information about running on agent (#2712)
puppetlabs/puppetdb: documentation/load_testing_tool.markdown

make sure to use trampoline, i.e. `lein trampoline run benchmark ...`
so that the tool can shut down and clean up normally.

### Running on an agent other than master

Running the PuppetDB benchmark tool on the master can cause resource constraints
which may significantly skew performance on the master. If you would like to run
the benchmark tool on an agent this can be achieved following the instructions
below.

* In /etc/puppetlabs/puppetdb/conf.d/jetty.ini in the [jetty] section, set:

>     * host = 0.0.0.0 (http access from all agents) or

Reviewer: ```suggestion
    * host = 0.0.0.0 # (http access from all agents)
```

>     * host = <agent ip address> (access from specific agent)

Reviewer: ```suggestion
    * host = <agent ip address> # (access from specific agent)
```

* Install java on the agent

> * In the config.ini file, make sure to set the port to the puppetdb port for

Reviewer: ```suggestion
* On the agent, in the config.ini file, set the port to the puppetdb port for
```
